cirq-web/cirq_ts/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "web",
+  "version": "0.12.0",
+  "description": "Web widgets for Cirq",
+  "main": "index.js",
+  "private": true,
+  "dependencies": {
+    "@types/looks-same": "^4.1.0",
+    "gts": "^3.1.0",
+    "live-server": "^1.2.1",
+    "tsc-watch": "^4.2.9",
+    "typescript": "^4.2.4"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.18",
+    "@types/jsdom": "^16.2.11",
+    "@types/mocha": "^8.2.2",
+    "@types/node": "^14.11.2",
+    "@types/pixelmatch": "^5.2.3",
+    "@types/pngjs": "^6.0.0",
+    "@types/puppeteer": "^5.4.3",
+    "@types/temp": "^0.9.0",
+    "@types/three": "^0.128.0",
+    "canvas": "^2.8.0",
+    "chai": "^4.3.4",
+    "concurrently": "^6.1.0",
+    "gts": "^3.1.0",
+    "install": "^0.13.0",
+    "jsdom": "^16.6.0",
+    "mocha": "^9.0.0",
+    "npm": "^7.16.0",
+    "nyc": "^15.1.0",
+    "pixelmatch": "^5.2.1",
+    "pngjs": "^6.0.0",
+    "puppeteer": "^10.0.0",
+    "sinon": "^11.1.1",
+    "temp": "^0.9.4",
+    "three": "^0.128.0",
+    "ts-loader": "^9.1.2",
+    "ts-mocha": "^8.0.0",
+    "ts-mock-imports": "^1.3.7",
+    "ts-node": "^10.0.0",
+    "typescript": "^4.0.3",
+    "webpack": "^5.37.0",
+    "webpack-cli": "^4.7.0",
+    "webpack-dev-server": "^4.0.0-beta.0"
+  },
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "npm test && webpack --mode production",
+    "test": "webpack --mode production && mocha -r ts-node/register 'e2e/**/*.ts' 'src/**/*_test.ts'",
```

Comment:
1. we want to separate unit tests from e2e tests, because unit tests are faster.
2. also, `webpack --mode production` can be replaced by `npm build`